components/demo/demo-page.js

```diff
@@ -59,11 +65,15 @@ class DemoPage extends LitElement {
 		return html`
 			<main>
 				<h1 class="d2l-heading-2">${this.pageTitle}</h1>
-				<div class="d2l-demo-page-content"><slot></slot></div>
+				<div class="d2l-demo-page-content" @fullscreen-toggle="${this._handleFullscreenToggle}"><slot></slot></div>
 			</main>
 		`;
 	}
 
+	_handleFullscreenToggle() {
+		this.classList.toggle('no-scroll');
```

Comment:
Another way to do this that doesn't modify the DOM outside of Lit's render cycle would be to set a `noScroll` property that reflects (as `no-scroll`), and change the CSS selector to be `[:host([no-scroll])]`.